taxi_service/urls.py

```diff
@@ -22,4 +22,5 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("", include("taxi.urls", namespace="taxi")),
+    path("registration/", include("django.contrib.auth.urls"))
```

Comment:
Consider using a more universal name like 'accounts/'.

Reply:
ok, I will change to "accounts/", but in our video lessons they showed to write "registration/"